src/properties.cpp

```diff
@@ -164,6 +164,8 @@ void Properties::loadSettings()
 
     windowMaximized = m_settings->value(QLatin1String("LastWindowMaximized"), false).toBool();
 
+    SwapMouseButtons2and3 = m_settings->value(QLatin1String("SwapMouseButtons2and3"), false).toBool();
```

Comment:
Coding style: the first character of a variable name is generally not capitalized.